x-pack/plugins/security/server/authentication/authenticator.test.ts

```diff
@@ -805,7 +806,7 @@ describe('Authenticator', () => {
 
       it('does not redirect to Access Agreement if authentication fails', async () => {
         const request = httpServerMock.createKibanaRequest();
-        mockOptions.session.get.mockResolvedValue(mockSessVal);
```

Comment:
I guess this was technically wrong before as well, but just not ever something that impacted the test?

Reply:
Yeh, a lot of these tests were copy pasted from previous tests which worked fine for what they were trying to assert but due to the extra logic we added we need to now make sure the session state is correct.